src/openrct2-ui/windows/About.cpp

```diff
@@ -91,25 +91,25 @@ static uint64_t window_about_page_enabled_widgets[] = {
     DEFAULT_ENABLED_WIDGETS,
 };
 
-static void window_about_openrct2_mouseup(rct_window *w, rct_widgetindex widgetIndex);
-static void window_about_openrct2_paint(rct_window *w, rct_drawpixelinfo *dpi);
-static void window_about_openrct2_invalidate(rct_window *w);
+static void WindowAboutOpenrct2Mouseup(rct_window *w, rct_widgetindex widgetIndex);
+static void WindowAboutOpenrct2Paint(rct_window *w, rct_drawpixelinfo *dpi);
+static void WindowAboutOpenrct2Invalidate(rct_window *w);
 
-static void window_about_rct2_mouseup(rct_window *w, rct_widgetindex widgetIndex);
-static void window_about_rct2_paint(rct_window *w, rct_drawpixelinfo *dpi);
-static void window_about_openrct2_common_paint(rct_window *w, rct_drawpixelinfo *dpi);
+static void WindowAboutRct2Mouseup(rct_window *w, rct_widgetindex widgetIndex);
```

Comment:
```suggestion
static void WindowAboutRCT2Mouseup(rct_window *w, rct_widgetindex widgetIndex);
```

See previous statement.